adm_program/installation/installation.php

```diff
@@ -685,6 +685,18 @@
                  , ('.$categoryIdSocialNetworks.', \'TEXT\', \'YAHOO_MESSENGER\',\'INS_YAHOO_MESSENGER\', NULL, \'yahoo.png\', NULL, 0, 8, '.$gCurrentUser->getValue('usr_id').',\''. DATETIME_NOW.'\') ';
     $db->query($sql);
 
+    // create user relation types
+    $sql = 'INSERT INTO '.TBL_USER_RELATION_TYPES.' (urt_id, urt_name, urt_name_male, urt_name_female, urt_id_inverse, urt_usr_id_create, urt_timestamp_create)
```

Comment:
It's definitely necessary to set the urt_id because it is needed in urt_id_inverse. Just check the following example:

```
urt_id     urt_name     urt_id_inverse
1          parent       2
2          child        1
3          sibling      3
```